test_jsonschema.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023-2024 Ericsson AB.
+# For a full list of individual contributors, please see the commit history.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import subprocess
+
+import jsonschema
+import pytest
+
+# Set up a console logger
+logger = logging.getLogger("__Logger__")
```

Comment:
Any particular reason to not use

```suggestion
logger = logging.getLogger(__name__)
```

as recommended by https://docs.python.org/3/library/logging.html?

> ... the recommended construction `logging.getLogger(__name__)`.

Reply:
Nope. I just copied that code from a code example and didn't reflect on that. Will actually remove the logger since it is anyway not used in the committed code.